security/keycloak-oauth2/src/test/java/io/quarkus/ts/security/keycloak/oauth2/OpenShiftRhSsoOauth2SecurityIT.java

```diff
@@ -1,24 +1,25 @@
 package io.quarkus.ts.security.keycloak.oauth2;
 
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM;
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM_BASE_PATH;
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM_FILE;
+
 import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
 import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
 
 import io.quarkus.test.bootstrap.KeycloakService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.OpenShiftScenario;
-import io.quarkus.test.services.Container;
+import io.quarkus.test.services.KeycloakContainer;
 import io.quarkus.test.services.QuarkusApplication;
 
 @OpenShiftScenario
 @DisabledIfSystemProperty(named = "ts.arm.missing.services.excludes", matches = "true", disabledReason = "https://github.com/quarkus-qe/quarkus-test-suite/issues/1145")
 @EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
 public class OpenShiftRhSsoOauth2SecurityIT extends BaseOauth2SecurityIT {
 
-    static final int KEYCLOAK_PORT = 8080;
-
-    @Container(image = "${rhsso.image}", expectedLog = "Http management interface listening", port = KEYCLOAK_PORT)
-    static KeycloakService keycloak = new KeycloakService(REALM_DEFAULT)
-            .withProperty("SSO_IMPORT_FILE", "resource::/keycloak-realm.json");
+    @KeycloakContainer(command = { "start-dev", "--import-realm" }, image = "${rhbk.image}")
```

Comment:
Why some of the command `KeycloakContainers` contains contain more commands that these two. When they were the same with using `Container`.

Reply:
I removed `.withProperty("SSO_IMPORT_FILE", "resource::/keycloak-realm.json");` so I want to import it from command. I can't answer it in general, if you deem particular change wrong, please comment on the very change and I'll check.

Reply:
Just that somewhere `@Conatiner` change to this
```
@KeycloakContainer(command = { "start-dev", "--import-realm", "--hostname-strict=false",
            "--features=token-exchange" }, image = "${rhbk.image}")
```
and somewhere to this
```
@KeycloakContainer(command = { "start-dev", "--import-realm" }, image = "${rhbk.image}")
```

But this was answered in other comment so now there is probably no need for some of these commands.
But as it was written in PR description let's solve these problem as the `start-dev` is solved.

Reply:
just note - I won't pretend I couldn't make mistake, but this is how I worked:

- if I remove import env var, I need to add `--import-realm`
- If I change RHSSO for RHBK it is change from Wildfly to Quarkus based KC, so I followed what is done in that very project for non-OCP tests. so typically if there were `--features=token-exchange` for baremetal tests I just added it as well. I didn't try to think whether it needs to be in baremetal tests at the first place.